styles/style.css

```diff
@@ -73,9 +83,7 @@
   /* Headline section */
   .headline {
     width: 100vw;
-    height: 88vh;
-    position: relative;
-    top: 60px;
+    height: 78vh;
     background-image: url(../images/header-illsutration-mobile@2x.png);
     background-size: cover;
     background-repeat: no-repeat;
```

Comment:
- [ ] kindly consider making sure that this section is not messed up as you build its desktop version to keep consistence with the provided design check how it was messed up below 
- your design 👇 
<img width="369" alt="Screen Shot 2023-05-10 at 08 41 15" src="https://github.com/mzirkof/R-PortFolio/assets/60146030/fd157177-57c6-43cb-9429-c918741232e5">


- Figma design 👇 
<img width="340" alt="Screen Shot 2023-05-10 at 08 44 41" src="https://github.com/mzirkof/R-PortFolio/assets/60146030/197e1ba5-fa46-4d7b-8985-0f39a13530fd">

Reply:
Ok done

Reply:
:clap:  Thanks @divinecharlotte for the review. I have done all the reviewed changes  :hammer: . Just hope that everything on the project meets the requirement :checkered_flag:

Reply:
@wayungi and @divinecharlotte  Please can I know more about this issue? because it seems like my browser isn't showing the same way at all. On my side, I am working with Firefox and I have tried the mobile version on all the responsive screen sizes of my browser and it shows well.